### PR TITLE
update: lens pkg

### PR DESCRIPTION
**Pull request by carlagn**
With the current leanest version of lens

### CODE REVIEW

**Comment by nilubava** [comment]
@carlagn Please dont delete the local fonts in docs. It is there until we figure out a better way to serve fonts

File: src/styles/layout.css [hunk 1]
```diff
@@ -1,83 +1,3 @@
-/** Inter Regular */
```
Comment (nilubava) [inline src/styles/layout.css]: Please do not remove the fonts here

File: src/styles/all.css [hunk 2]
```diff
@@ -0,0 +1,944 @@
+.fa {
```
Comment (nilubava) [inline src/styles/all.css]: Are these icons used in docs homepage from the pro package? Then, I think we should not expose these icons in this public repo. We can load these icons from Lens

File: src/styles/layout.css (hunk 1 above)
Comment (carlagn) [reply src/styles/layout.css]: they come from lens though? this is a repeated import

File: src/styles/all.css (hunk 2 above)
Comment (carlagn) [reply src/styles/all.css]: I tried it without the icons from lens and it didn't load them :/

File: src/styles/layout.css (hunk 1 above)
Comment (nilubava) [reply src/styles/layout.css]: We had to keep the fonts local to avoid the loading issue.

**Pull request merged**